libs/island-ui/core/src/lib/Filter/FilterInput/FilterInput.tsx

```diff
@@ -0,0 +1,42 @@
+import { theme } from '@island.is/island-ui/theme'
+import React, { useEffect, useState } from 'react'
+import { useWindowSize } from 'react-use'
+import { Input } from '../../Input/Input'
+
+export interface FilterInputProps {
+  name: string
+  id?: string
+  placeholder?: string
+  value: string
+  onChange: (value: string) => void
+}
+
+export const FilterInput: React.FC<FilterInputProps> = ({
+  name,
+  id = name,
+  placeholder = '',
+  value = '',
+  onChange,
+}) => {
+  const { width } = useWindowSize()
+  const [isMobile, setIsMobile] = useState(false)
+  useEffect(() => {
+    if (width < theme.breakpoints.md) {
+      return setIsMobile(true)
+    }
+    setIsMobile(false)
+  }, [width])
```

Comment:
Maybe we should implement this the same way we do with FilterMultiChoice using responsive displays. Or we could possibly implement responsive background for the input component

Reply:
Yes I was actually about doing the background responsive on the input as it would save the additional responsive display markup.